### PR TITLE
EWL-6731 Fixes people listing page with uneven number cards 

### PR DESCRIPTION
**Pull request by designerneil**
## Ticket(s)

**Jira Ticket**
- [EWL-6731: People Listing – People Listing - Stacking Bug](https://issues.ama-assn.org/browse/EWL-6730)

## Description
People listing page with uneven number of cards centers the last row of cards when they should be justified left.

## To Test
- [ ] run `gulp serve`
- [ ] enable local SG2 in your D8 environment
- [ ] clear cache `drush @one.local cr`
- [ ] visit http://ama-one.local/member-groups-sections/integrated-physician-practices/integrated-physician-practice-section-ipps
- [ ] observe the last row is left justified
- [ ] visit http://localhost:3000/?p=pages-people-listing
- [ ] observe the last row is left justified
- [ ] Did you test in IE 11?

## Visual Regressions
A report is available [here](http://htmlpreview.github.io/?https://github.com/AmericanMedicalAssociation/ama-style-guide-2/blob/visual-regression-testing-artifact/feature/EWL-6731-people-stacking/html_report/index.html).

## Relevant Screenshots/GIFs
screen shot 2019-02-12 at 2 17 27 pm

## Remaining Tasks
n/a

## Additional Notes
n/a
---

[Guidelines for Contribution](CONTRIBUTING.md)
[SG2 Standards](ama-style-guide-2/docs/standards.md)

### CODE REVIEW

**Comment by JaeBea** [comment]
@designerneil ,
Please update description to point to the correct ticket.

**Review by JaeBea — CHANGES_REQUESTED** [review]
@designerneil ,
This solution does not correct the core issue. It accounts only for when there are 3 orphan listings. A similar issue was addressed for the subcat exploration blocks. The code for it can be found in `subcategory.js`. With some tweaks this should be able to be applied to this issue as well. Please let me know if you have questions about how that solution was implemented.

**Comment by designerneil** [comment]
@JaeBea thank you for your review. I looked at `subcategory.js` and found the javascript to be very complicated and decided to write my own to get the cards line up left when there is an odd number of them. I removed the flexbox and used inline-block instead.

File: styleguide/source/assets/scss/05-pages/_people-listing.scss [hunk 1]
```diff
@@ -6,16 +6,24 @@
   &__card-container {
     @include gutter($padding-top-full...);
     @include gutter($padding-bottom-full...);
-    display: flex;
-    flex-wrap: wrap;
+
```
Comment (JaeBea) [inline styleguide/source/assets/scss/05-pages/_people-listing.scss]:
```suggestion
```

File: styleguide/source/assets/scss/05-pages/_people-listing.scss [hunk 2]
```diff
@@ -6,16 +6,24 @@
...
+      margin-left: calc(-12.08% * .1);
+      margin-right: calc(-12.08% * .1);
+    }
+
+    @include breakpoint($bp-large) {
+      margin-left: calc(-22.08% * .1);
+      margin-right: calc(-22.08% * .1);
+    }
+
+    &:after {
```
Comment (JaeBea) [inline styleguide/source/assets/scss/05-pages/_people-listing.scss]: Remove `&:after` rule. With change towards inline-block layout, this is not needed.

**Review by JaeBea — CHANGES_REQUESTED** [review]
@designerneil ,
There are two changes noted below before this can be merged. They should be very quick to fix.

**Comment by designerneil** [comment]
@JaeBea thanks! Removed pseudo class

File: styleguide/source/assets/scss/05-pages/_people-listing.scss (hunk 1 above)
Comment (JaeBea) [reply styleguide/source/assets/scss/05-pages/_people-listing.scss]: @designerneil can you accept this code styling change?

Comment (designerneil) [reply styleguide/source/assets/scss/05-pages/_people-listing.scss]: @JaeBea I don't understand why those need to get removed?

Comment (JaeBea) [reply styleguide/source/assets/scss/05-pages/_people-listing.scss]: @designerneil The extra space?

**Review by JaeBea — APPROVED** [review]
@designerneil ,
I see that the requested changes have been made. Good catch with adding the Drupal wrapper. I will approve and merge into develop.

**Pull request merged**